src/storage/documentService/documentService.js

```diff
@@ -68,7 +69,7 @@ const uploadDocumentAtId = async (
     throw new Error("Ttl cannot exceed 90 days");
   }
 
-  const fragments = await verify({ network: config.network })(document);
+  const fragments = await verify(document);
   if (!isValid(fragments)) {
```

Comment:
if verify returns `undefined`, and hence `isValid(undefined)`, this error will occur, is that expected?:

```
  if (verificationFragments.length < 1) {
                            ^

TypeError: Cannot read properties of undefined (reading 'length')
```

Reply:
[Refer to above comment]